newspaper/source.py

```diff
@@ -133,7 +129,39 @@ def set_feeds(self):
         """Don't need to cache getting feed urls, it's almost
         instant with xpath
         """
-        urls = self.extractor.get_feed_urls(self.url, self.categories)
+        common_feed_urls = ['/feed', '/feeds', '/rss']
```

Comment:
Great heuristics to catch extra feeds. Thanks for this. I'll probably test this new code and add unit tests for the new edge case catches later. Nice job